cmd/netmaker.go

```diff
@@ -131,10 +120,25 @@ func getLatestRelease() string {
 
 func testConnection() {
 	pterm.Println("\nTesting Server setup for https://api." + domain + "/api/server/status")
+	for i := 1; i < 9; i++ {
```

Comment:
could perhaps bind the value (9) to a variable and then later on refer to variable-1